test/slp_server_test.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SlpServer"
+
+RSpec.describe "Yast::SlpServer" do
```

Comment:
Not good a good reason. IIRC, in recent versions of `rspec-rails`, that form was used and, you know, I got used to it :)